pkc

```diff
@@ -296,16 +296,43 @@ run_setup() {
     echo "Done!"
 }
 
-restore_all() {
-    archive_file=$1
-    # Extract tarball and restore images
-    docker exec -it micro-pkc-mediawiki /MediaWiki_Backup/extract_and_restore_images.sh -a "/backup/$archive_file" -w /var/www/html
-    # Restore database
-    docker exec -it micro-pkc-database /MediaWiki_Backup/restore_database.sh /backup/database.sql.gz
-    # Clean up database sql file
-    docker exec -it micro-pkc-database rm /backup/database.sql.gz
-    # Toggle read-only off
-    docker exec -it micro-pkc-mediawiki /MediaWiki_Backup/toggle_read_only_off.sh /var/www/html
+create_new_backup() {
+    sqlfile=$(docker run --rm --volumes-from micro-pkc-mediawikibackup busybox \
+        find /backup/latest -name '*.sql.gz' | head -1)
+    imgfile=$(docker run --rm --volumes-from micro-pkc-mediawikibackup busybox \
+        find /backup/latest -name '*.img.tar*' | head -1)
+
+    if [[ -n "$sqlfile" && -n "$imgfile" ]]; then
+        echo "Saving a copy of the most recent backup..."
+        tmpdir=$(mktemp -d || exit 1)
+        docker cp micro-pkc-mediawikibackup:"$sqlfile" \
+            "$tmpdir/$(basename "$sqlfile")"
+        docker cp micro-pkc-mediawikibackup:"$imgfile" \
+            "$tmpdir/$(basename "$imgfile")"
+        tar -cf "backup-$(date +"%Y-%m-%d_%Hh%Mm.%A")".tar -C "$tmpdir" .
```

Comment:
On iteration I would love to see <domain_id><$wgserver / DNS name>

Reply:
noted. doing so will benefit the ability to vertically scale the autobackup service